### ISSUE TITLE
Bug: Infinite lore lines possible if not using the command /setloreline

### ISSUE THREAD

**Issue by HorusApl** [JustBru00/RenamePlugin#193]
In the config, there is a limit of max lore lines for the command /setloreline but the other commands like /addlore or /addloreline for some reason do not respect this limit when they absolutely should.

**Comment by JustBru00** [comment]
This feature is working as intended.
`setloreline_max_lines` only sets the maximum number of lore lines for /setloreline.

Would you like an option to set the maximum number of lore lines for /addlore and /addloreline?

**Comment by HorusApl** [comment]
Yes please, if this only works with one command then the original intent behind the feature I think is negated.
